test/integration2/dialects/postgresql.spec.js

```diff
@@ -0,0 +1,119 @@
+const { expect } = require('chai');
+const {
+  getAllDbs,
+  getKnexForDb,
+  Db,
+  getDbTestConfig,
+} = require('../util/knex-instance-provider');
+
+describe('PostgreSQL dialect', () => {
+  const dbs = getAllDbs().filter((db) => db === Db.PostgresSQL);
```

Comment:
so uh, why do you get a list and filter it down to a single value, instead of just starting with that single value??

Reply:
good point, I was mostly just repeating this procedure the same way it was done in other integration tests

Reply:
@kibertoad so should I just fix this part and then this is ready for merge?

Reply:
let's try enabling tests for CockroachDB as well and see if that would also pass